### ISSUE TITLE
turn surport ? 

### ISSUE THREAD

**Issue by notedit** [rawrtc/rawrtc#9]
any plan for turn surport

**Comment by lgrahl** [comment]
Yes and high priority as well. It will be the next thing to be added after resolving STUN server URLs.

**Comment by lgrahl** [comment]
Let me correct that: Next thing after there is basic support for the WebRTC API (and the offer/answer procedure - see #18). I hope you don't mind but I think this is the most crucial feature for most people as (sadly) pretty much nobody supports ORTC atm. :)

**Comment by lgrahl** [comment]
@notedit wanted to let you know that TURN support is nearing completion (see branch `turn`). Just some bugs that need to be fixed.
